src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -365,6 +367,14 @@
     </xsd:restriction>
   </xsd:simpleType>
 
+  <xsd:simpleType name="key_type">
+    <xsd:restriction base="xsd:string">
+      <xsd:enumeration value="string" />
```

Comment:
not sure this is useful
ppl will want "int" just, but PHP always auto-casts so this is useless if there's no processing